typings/index.d.ts

```diff
@@ -206,12 +216,13 @@ declare module 'ytdl-core' {
       published: number;
       video_url: string;
       age_restricted: boolean;
-      likes?: number;
-      dislikes?: number;
+      likes: number | null;
+      dislikes: number | null;
       media: Media;
       author: Author;
       thumbnails: thumbnail[];
       storyboards: storyboard[];
+      description: string | null;
```

Comment:
this goes in `MoreVideoDetails`

Reply:
it's also not `| null`, is it? the property is based on `shortDescription: string`

Reply:
After testing, if there's no description, the `description` field is `null`.

Reply:
> this goes in `MoreVideoDetails`

It's already there

Reply:
it is? i don't see it. this property is retrieved from the raw `videoDetails` btw

https://github.com/fent/node-ytdl-core/blob/master/lib/info-extras.js#L265

Reply:
It being `null` makes sense, 
```js
videoDetails.description = videoDetails.shortDescription || getText(videoDetails.description);
```
when there's no description, `shortDescription` is `""`, and that is a falsy value, so it assigns whatever is returned from the second part into `videoDetails.description`, if there's no description there isn't anything other than `shortDescription` in the `info`, so the function `getText` returns null in this case.
The expression `??` solves this problem because it moves to the next part if the first is `null` or `undefined`, but that would require node 14 at least.

Reply:
We could leave it like this or change the assignment to return empty string, what do you think?
sth like this:
```js
videoDetails.description = videoDetails.shortDescription || getText(videoDetails.description) || '';
```

Reply:
i agree with it being `null`. but i still don't see how it's available in the raw `VideoDetails`. it's in `MicroformatRenderer`, and then in `MoreVideoDetails` after being added from the above line

Reply:
I'm not sure if I understand you correctly, as far as I can see, it's within `MoreVideoDetails`, unless you're talking about something else
![Capture](https://user-images.githubusercontent.com/23695024/104084244-93518800-5299-11eb-85a2-ace7b9c452b7.PNG)

Reply:
oh! my mistake. i didn't look at the diff correctly and thought it was below `VideoDetails`. sorry